src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -33,6 +32,7 @@
     PNDMScheduler,
 )
 from ...utils import deprecate, logging
+from ..pipeline_utils import DiffusionPipeline
```

Comment:
```suggestion
from .. import DiffusionPipeline
```

(I think) Or maybe `from ...pipelines import DiffusionPipeline`

Reply:
I think here we should import from the "shortest" path to avoid circular imports, so think this is correct :-)

Reply:
You are right, I didn't think about that.